<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>730</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/image/icons/th.jpg</normaloff>:/image/icons/th.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>380</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>60</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/icons/4-Code Sand.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>41</width>
       <height>41</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>170</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/icons/pic.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>280</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/icons/video.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>570</y>
      <width>651</width>
      <height>81</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_photo">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>640</width>
        <height>70</height>
       </rect>
      </property>
      <property name="title">
       <string>文字提取</string>
      </property>
      <widget class="QPushButton" name="pushButton_choose">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>选择图片</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_cut">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>图片裁剪</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ocr">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>文字提取</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_back">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>20</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/icons/back.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_picture">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>640</width>
        <height>70</height>
       </rect>
      </property>
      <property name="title">
       <string>图片处理</string>
      </property>
      <widget class="QPushButton" name="pushButton_grey">
       <property name="geometry">
        <rect>
         <x>125</x>
         <y>30</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>灰度图像</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_soft">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>平滑图像</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_hard">
       <property name="geometry">
        <rect>
         <x>355</x>
         <y>30</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>锐化图像</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_cut1">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>30</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>图像裁剪</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_back1">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>30</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/icons/back.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_choose1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>选择图像</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_video">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>640</width>
        <height>70</height>
       </rect>
      </property>
      <property name="title">
       <string>文档扫描</string>
      </property>
      <widget class="QPushButton" name="pushButton_video">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>打开摄像头</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_video_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>拍照</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_toocr">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>扫描</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>941</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>481</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>30</y>
        <width>281</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>411</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border: 1px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>0</y>
        <width>411</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border: 1px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QVideoWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>461</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>选择摄像头</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_reload">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icon.qrc">
        <normaloff>:/image/icons/reload.png</normaloff>:/image/icons/reload.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>80</y>
        <width>391</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>      文件      </string>
    </property>
    <widget class="QMenu" name="menuSave_as">
     <property name="title">
      <string>Save as</string>
     </property>
     <addaction name="actionJPG"/>
     <addaction name="actionPNG"/>
     <addaction name="actionTIFF"/>
    </widget>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="menuSave_as"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menu_type">
    <property name="title">
     <string>      类型      </string>
    </property>
    <addaction name="action_Photo"/>
    <addaction name="action_Picture"/>
    <addaction name="action_Video"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_type"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/image/icons/add.png</normaloff>:/image/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/image/icons/save.png</normaloff>:/image/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Picture">
   <property name="icon">
    <iconset>
     <normaloff>:/icons_image/icons/pic.png</normaloff>:/icons_image/icons/pic.png</iconset>
   </property>
   <property name="text">
    <string>Picture</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/image/icons/close.png</normaloff>:/image/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Photo">
   <property name="icon">
    <iconset>
     <normaloff>:/icons_image/icons/usercenter.png</normaloff>:/icons_image/icons/usercenter.png</iconset>
   </property>
   <property name="text">
    <string>Photo</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Video">
   <property name="icon">
    <iconset>
     <normaloff>:/icons_image/icons/video.png</normaloff>:/icons_image/icons/video.png</iconset>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionJPG">
   <property name="text">
    <string>JPEG</string>
   </property>
  </action>
  <action name="actionPNG">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionTIFF">
   <property name="text">
    <string>TIFF</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Close</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Photo</sender>
   <signal>triggered()</signal>
   <receiver>pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Picture</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Video</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_3</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
